src/ut/baseresolver_test.cpp

```diff
@@ -140,7 +151,7 @@ TEST_F(BaseResolverTest, IPv4AddressResolutionManyTargets)
 TEST_F(BaseResolverTest, ARecordAtLeastOneGray)
 {
   add_white_records(DEFAULT_COUNT);
-  AddrInfo gray_record = ip_to_addr_info("3.0.0.0");
+  AddrInfo gray_record = ResolverTest::ip_to_addr_info("3.0.0.0");
```

Comment:
I have changed the method to be static, as there's no need for a ResolverTest object.